document-legacy/src/layers/layer_info.rs

```diff
@@ -119,7 +119,7 @@ impl<'a> TryFrom<&'a Layer> for &'a Subpath {
 
 /// Defines shared behavior for every layer type.
 pub trait LayerData {
-	/// Render the layer as an SVG tag to a given string.
+	/// Render the layer as an SVG tag to a given string, returning if rendering fully successful.
```

Comment:
I'm having trouble understanding this added part for the return value. Would you mind fixing the grammar and elaborating a bit on what it means to have rendered "fully" successfully? I can't tell if "fully" means the full area so no parts were culled, or if it means that some parts may have failed somehow, or what else that might mean.